ciao-4.10/contrib/bin/srcflux

```diff
@@ -313,7 +313,13 @@ def save_regions( myparams, regions, flavor ):
     """    
     from ciao_contrib.runtool import dmmakereg
     from tempfile import NamedTemporaryFile    
-    ss = stk.build(regions)
+
+    # CIAO 4.10 changes to NULL regions no longer creates a fake
+    # "point(0,0)" -- I'm using that so we'll put it back.
+    try:
+        ss = stk.build(regions)
+    except ValueError as ve:
```

Comment:
In general I'm a bit suspicious of code catching such a generic error - i.e. what happens if there's an error thrown because of a case other than "is empty" - e.g. perhaps the file is corrupted or been eaten by a gru. It's not obvious from a whole 10 seconds of thinking about it what a cleaner way to handle this case is though.

I guess the fact you read from the file below catches the case of "there's something borked with the file" case.